curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/614090d5a22b6f0a5a6b464c.md

```diff
@@ -9,7 +9,11 @@ dashedName: step-13
 
 # --description--
 
-Use the `>` selector to target the unordered list elements within the `nav` elements, and use _Flexbox_ to evenly space the children.
+The child combinator selector `>` is used between selectors to target only elements that match the second selector and are a direct child of the first selector.
+
+This can be helpful when you have deeply nested elements and want to control the scope of your styling.
+
+Use the `>` selector to target the unordered list elements within the `nav` elements, and use Flexbox to evenly space the children.
```

Comment:
It looks like the original here had the word Flexbox italicized so we should probably keep that part that same.

```suggestion
Use the `>` selector to target the unordered list elements within the `nav` elements, and use _Flexbox_ to evenly space the children.
```

Reply:
Agreed - thanks @jdwilkin4!